SPECS/coredns/coredns-1.6.7.signatures.json

```diff
@@ -1,6 +1,6 @@
 {
  "Signatures": {
-  "coredns-1.6.7-vendor.tar.gz": "3d388da75cdb9db6ee892c44e5bbe2af0e38ec7c757c6936503672e8d990b155",
+  "coredns-1.6.7-vendor.tar.gz": "b66685ad4546b80920fd97b124194eeaf121784787e4d06d22af2754b933d6de",
```

Comment:
Should we be bumping the release numbers on these, on account of the new vendor tarballs?

Reply:
Resolved offline